buildSrc/build.gradle

```diff
@@ -0,0 +1,21 @@
+plugins {
+    id 'groovy-gradle-plugin'
+}
+
+repositories {
+    gradlePluginPortal()
+}
+
+wrapper {
+    distributionType = Wrapper.DistributionType.ALL
+    gradleVersion = "7.0.2"
+}
+
+dependencies {
+    implementation group: 'nebula.lint', name: 'nebula.lint.gradle.plugin', version: '16.23.0'
```

Comment:
the versions of the plugins are set as normal dependencies in the building of the plugin